controllers/configurationpolicy_controller.go

```diff
@@ -2124,15 +2114,11 @@ func (r *ConfigurationPolicyReconciler) createParentPolicyEvent(instance *policy
 }
 
 func createParentPolicy(instance *policyv1.ConfigurationPolicy) extpoliciesv1.Policy {
-	ns := common.ExtractNamespaceLabel(instance)
-	if ns == "" {
-		ns = NamespaceWatched
-	}
-
 	return extpoliciesv1.Policy{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      instance.OwnerReferences[0].Name,
-			Namespace: ns, // we assume that the parent policy is in the watched-namespace passed as flag
+			Name: instance.OwnerReferences[0].Name,
+			// It's assumed that the parent policy is in the same namespace as the configuration policy
```

Comment:
BTW, this is a *very* safe assumption. From https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/#owner-references-in-object-specifications:
> A valid owner reference consists of the object name and a UID within the same namespace as the dependent object.